src/AppInstallerCLICore/Workflows/ConfigurationFlow.cpp

```diff
@@ -96,36 +96,84 @@ namespace AppInstaller::CLI::Workflow
             return {};
         }
 
-        void OutputValueSet(OutputStream& out, const ValueSet& valueSet, size_t indent)
+        void OutputPropertyValue(OutputStream& out, const IPropertyValue property)
+        {
+            switch (property.Type())
+            {
+            case PropertyType::String:
+                out << ' ' << Utility::ConvertToUTF8(property.GetString()) << '\n';
+                break;
+            case PropertyType::Boolean:
+                out << ' ' << (property.GetBoolean() ? "true" : "false") << '\n';

```

Comment:
These boolean concept names should probably be localized... c'est vrais.

Reply:
ei